sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -265,6 +267,8 @@ private static void AssertMultipleDispatchLogs(IHost host)
 
             Assert.True(logMessages.Where(x => !string.IsNullOrEmpty(x.FormattedMessage)
                 && x.FormattedMessage.Contains("Sending events to EventHub")).Count() > 0);
```

Comment:
nit:  The `Count()` calls on 166 and 269 could be `Any()` since the assertion is just `> 0`